id/crypto.go

```diff
@@ -153,3 +153,13 @@ type CrossSigningKey struct {
 	Key   Ed25519
 	First Ed25519
 }
+
+// Secret storage keys
+type Secret string
+
+const (
+	SecretXSMaster       Secret = "m.cross_signing.master"
+	SecretXSSelfSigning  Secret = "m.cross_signing.self_signing"
+	SecretXSUserSigning  Secret = "m.cross_signing.user_signing"
+	SecretMegolmBackupV1 Secret = "m.megolm_backup.v1"
```

Comment:
These values should probably be deduplicated with the account data type, probably by referencing these constants in the account data types (as in `AccountDataCrossSigningMaster = Type{string(id.SecretXSMaster), AccountDataEventType}`)